src/_lib/modules/charts/chart.component.ts

```diff
@@ -46,7 +46,7 @@ export class ChartComponent implements OnInit, AfterViewInit {
     @ViewChild('hoverOverlay') hoverOverlay: ElementRef;
 
     @Output() nwMousemove: EventEmitter<[number, number]> = new EventEmitter();
-    @Output() nwMouseout: EventEmitter<null> = new EventEmitter();
+    @Output() nwMouseleave: EventEmitter<null> = new EventEmitter();
```

Comment:
I'm ok with changing the name of this - but you'll need to update all references to this output in Spike. This change is a breaking change so will also require a new major version